distributed/semaphore.py

```diff
@@ -271,21 +277,28 @@ def __init__(self, max_leases=1, name=None, client=None):
         self.name = name or "semaphore-" + uuid.uuid4().hex
         self.max_leases = max_leases
 
-        if self.client.asynchronous:
-            self._started = self.client.scheduler.semaphore_create(
-                name=self.name, max_leases=max_leases
-            )
-        else:
-            self.client.sync(
-                self.client.scheduler.semaphore_create,
+        self._started = None
```

Comment:
What would you think about renaming this to something like `self._register`? I'm not sure if `self._started` is still meaningful here

Reply:
Or perhaps `_registered`, to indicate that `register=True` had completed successfully?